land.arcana.TarotCanvas.yml

```diff
@@ -0,0 +1,33 @@
+app-id: land.arcana.TarotCanvas
+runtime: org.kde.Platform
+runtime-version: '6.10'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '6.10'
+command: tarot-canvas
+finish-args:
+  - --share=network                       # Needed to download reference decks
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+modules:
+  - python3-modules.yaml
+  - name: tarot-canvas
+    buildsystem: simple
+    build-commands:
+      - |
+        PYTHON_VERSION=$(python3 -c "import sys; print(f'{sys.version_info.major}.{sys.version_info.minor}')") && \
+        SITE_PACKAGES="${FLATPAK_DEST}/lib/python${PYTHON_VERSION}/site-packages" && \
+        mkdir -p ${SITE_PACKAGES} && \
+        cp -r tarot_canvas ${SITE_PACKAGES}/
```

Comment:
you should use a proper python buildsystem instead of hacking manual install commands.